src/main/java/org/marketdesignresearch/mechlib/core/bidder/newstrategy/BoundValueQueryWithMRPARRefinementStrategy.java

```diff
@@ -3,10 +3,11 @@
 import java.util.Set;
 
 import org.marketdesignresearch.mechlib.core.bid.bundle.BundleBoundValueBid;
+import org.marketdesignresearch.mechlib.mechanism.auctions.Auction;
 import org.marketdesignresearch.mechlib.mechanism.auctions.interactions.BoundValueQueryWithMRPARRefinement;
 
 public interface BoundValueQueryWithMRPARRefinementStrategy extends InteractionStrategy{
-	BundleBoundValueBid applyBoundValueQueryWithMRPARRefinementStrategy(BoundValueQueryWithMRPARRefinement query);
+	BundleBoundValueBid applyBoundValueQueryWithMRPARRefinementStrategy(BoundValueQueryWithMRPARRefinement query, Auction<?> auction);

```

Comment:
The Strategy now receives an Auction instance as well, when called to answer a certain query.